queue_services/entity-filer/tests/unit/test_worker/test_correction_bcia.py

```diff
@@ -500,43 +512,43 @@ async def test_worker_director_name_and_address_change(app, session, mocker, tes
 @pytest.mark.parametrize(
     'test_name',
     [
-        'ben_add_resolution_dates',
-        'ben_update_existing_resolution_dates',
-        'ben_update_with_new_resolution_dates',
-        'ben_delete_resolution_dates',
+        'bc_add_resolution_dates',
+        'bc_update_existing_resolution_dates',
+        'bc_update_with_new_resolution_dates',
+        'bc_delete_resolution_dates',
```

Comment:
can we update this test to test all the relevant entity types? i.e. BEN, BC, ULC, CC